lib/profile/index.js

```diff
@@ -66,7 +66,9 @@ function defaults ( ) {
     //, maxRaw: 200 // highest raw/noisy CGM value considered safe to use for looping
     , calc_glucose_noise: false
     , target_bg: false // set to an integer value in mg/dL to override pump min_bg
-  };
```

Comment:
Is this ; needed here?

Reply:
I didn't mean to change this line, but it doesn't make a difference; semicolons are optional in JS with only rare exceptions.